app/experimenter/experiments/api/v3/serializers.py

```diff
@@ -87,25 +89,30 @@ def validate(self, data):
         return validated_data
 
     def create(self, validated_data):
+        preset_data = NIMBUS_DATA["ExperimentDesignPresets"]["empty_aa"]["preset"][
+            "arguments"
+        ].copy()
+        audience_data = NIMBUS_DATA["Audiences"][validated_data["audience"]]
+
         validated_data.update(
             {
                 "slug": slugify(validated_data["name"]),
                 "owner": self.context["request"].user,
-                "firefox_channel": Experiment.CHANNEL_RELEASE,
+                "firefox_channel": audience_data["firefox_channel"],
+                "proposed_duration": preset_data["proposedDuration"],
+                "proposed_enrollment": preset_data["proposedEnrollment"],
             }
         )
         experiment = super().create(validated_data)
 
-        ExperimentVariant.objects.create(
-            experiment=experiment,
-            name="control",
-            slug="control",
-            ratio=1,
-            is_control=True,
-        )
-        ExperimentVariant.objects.create(
-            experiment=experiment, name="treatment", slug="treatment", ratio=1,
-        )
+        for branch_data in preset_data["branches"]:
+            ExperimentVariant.objects.create(
+                experiment=experiment,
+                name=branch_data["slug"],
```

Comment:
Q: should this be `branch_data["slug"]` or `"name"`?
I believe this is correct, per https://github.com/mozilla/nimbus-shared/blob/ecf090a942e29d76ce73a0d84d542cc91edf2398/data/ExperimentDesignPresets.ts#L31-L34 (if I'm looking at the correct lines); but just wanted to confirm.

Reply:
Yep I filed: https://github.com/mozilla/nimbus-shared/issues/95